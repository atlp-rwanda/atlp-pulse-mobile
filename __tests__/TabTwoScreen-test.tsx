import { render } from '@testing-library/react-native';

import TabOTwoScreen,  { CustomText }  from '@/app/(tabs)/two';
import React from 'react';

describe('<TabOTwoScreen />', () => {
  test('Text renders correctly on TabOTwoScreen', () => {
    const { getByText } = render(<TabOTwoScreen />);

    getByText('Tab Two');
  });

  test('CustomText renders correctly', () => {
    const tree = render(<CustomText>Some text</CustomText>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
