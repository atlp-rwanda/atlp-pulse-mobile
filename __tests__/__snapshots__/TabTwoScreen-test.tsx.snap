// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabOTwoScreen /> CustomText renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Some text
</Text>
`;

exports[`<TabOneScreen /> CustomText renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Some text
</Text>
`;
